interface/globals.php

```diff
@@ -565,24 +565,10 @@ function GetCallingScriptName()
 //    -if you don't want it translated, then strip the xl function away
 $tback = xl('(Back)');
 
-$version = (new VersionService())->fetch();
-
-if (!empty($version)) {
-    //Version tag
-    $patch_appending = "";
-    //Collected below function call to a variable, since unable to directly include
-    // function calls within empty() in php versions < 5.5 .
-    $version_getrealpatch = $version['v_realpatch'];
-    if (($version['v_realpatch'] != '0') && (!(empty($version_getrealpatch)))) {
-        $patch_appending = " (" . $version['v_realpatch'] . ")";
-    }
-
-    $openemr_version = $version['v_major'] . "." . $version['v_minor'] . "." . $version['v_patch'];
-    $openemr_version .= $version['v_tag'] . $patch_appending;
-} else {
-    $openemr_version = xl('Unknown version');
-}
-$GLOBALS['openemr_version'] = $openemr_version;
+$versionSql = "SELECT * FROM `version`";
+$versionRes = sqlQuery($versionSql);
+$versionString = "{$versionRes['v_major']}.{$versionRes['v_minor']}.{$versionRes['v_patch']}";
+$GLOBALS['openemr_version'] = $versionString;
```

Comment:
this is not incorporating the `v_tag` or `v_realpatch`, when applicable (see above code)
would also use sqlQueryNoLog() function since will be called on every globals and does not need to be logged

Reply:
Would it be possible to drop this completely from globals.php (like you planned to do) and use the VersionService in the patient portal instead?

Reply:
just realize there are several other places. I could probably make quick work of this (migrate all use to VersionService so do not need to build in global.php); will try to make a PR tonight for this.

Reply:
bye bye $GLOBALS['openemr_version'] :)
https://github.com/openemr/openemr/pull/4809